src/screens/CreatePost/index.js

```diff
@@ -55,11 +55,7 @@ import {Analytics} from '../../libraries/analytics/firebaseAnalytics';
 import {Button, ButtonAddMedia} from '../../components/Button';
 import {COLORS} from '../../utils/theme';
 import {Context} from '../../context';
-import {
-  DEFAULT_TOPIC_PIC_PATH,
-  MAX_POLLING_ALLOWED,
-  MIN_POLLING_ALLOWED
-} from '../../utils/constants';
+import {MAX_POLLING_ALLOWED, MIN_POLLING_ALLOWED} from '../../utils/constants';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the unused import `DEFAULT_TOPIC_PIC_PATH` is a good practice as it helps to keep the code clean and maintainable.
```diff
- import {DEFAULT_TOPIC_PIC_PATH, MAX_POLLING_ALLOWED, MIN_POLLING_ALLOWED} from '../../utils/constants';
+ import {MAX_POLLING_ALLOWED, MIN_POLLING_ALLOWED} from '../../utils/constants';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->